pkgs/by-name/um/umockdev/package.nix

```diff
@@ -65,7 +70,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   buildInputs = [
     glib
-    systemdMinimal
+    (if withSystemd then systemdMinimal else udev)
```

Comment:
Still looking for an answer to the question I asked in the previous submission: https://github.com/NixOS/nixpkgs/pull/405551#discussion_r2083252970